src/main/java/seedu/address/logic/commands/FilterTagCommand.java

```diff
@@ -6,18 +6,21 @@
  * Filters and lists all persons in address book who are tagged by any of the argument keywords.
  * Keyword matching is case-insensitive.
  */
-public class FilterTagCommand extends Filter {
+public class FilterTagCommand extends FilterCommand {
+
+    public static final String TYPE = "tag";
+
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters for all persons whose tags include any of "
             + "the specified keywords and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + "student";
+            + "Example: " + COMMAND_WORD + " tag student";
```

Comment:
Minor nit but maybe it would be better use the constant you just added.

```suggestion
            + "Example: " + COMMAND_WORD + " " + TYPE + " student";
```